lemon-demo-reactive/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
+services:
+
+  database-test:
+    image: mongo:3.2.20
```

Comment:
This is the latest image?

Reply:
No, I thought we should use one exact version.
Actually, we can use the tag `latest`, so whenever we build the container we will get the latest version of mongodb.